politeiawww/api/v1/api.md

```diff
@@ -1473,6 +1475,129 @@ Reply:
 }
 ```
 
+### `Proposal voting status`
+
+Returns the voting status for a single public proposal
+
+**Route:** `GET /V1/proposals/{token}/votingstatus`
+
+**Params:** none
+
+**Result:**
+
+| | Type | Description |
+|-|-|-|
+| proposalvoting | VotingStatus  | Proposal voting status |
+
+**VotingStatus:**
+
+| | Type | Description |
+|-|-|-|
+| token | string  | Censorship token |
+| status | int | Status identifier |
+| votessummary | CastedVotesSummary | Summary of casted votes
+
+**CastedVotesSummary:**
+| | Type | Description |
+|-|-|-|
+| totalvotes | int  | Total votes received |
+| optionsresult | voteoptionresult | Options info and votes received by each one |
+
```

Comment:
Do we need blockstart/end/hash here?

Reply:
No, this information won't be used by the GUI.